database/src/main/scala/cromwell/database/liquibase/DiffResultFilter.scala

```diff
@@ -159,5 +171,7 @@ object DiffResultFilter {
       * @return The updated diff result.
       */
     def filterChangedObjects(filters: Seq[DiffFilter]) = filter(diffResult, filters, (_, _) => false)
+
+    def filterOldeObjects = filter(diffResult, Seq.empty, isOldeObject)
```

Comment:
I thought that was covered by #789 but if not then there should certainly be a ticket for it.